calico.go

```diff
@@ -339,16 +367,6 @@ func cmdDel(args *skel.CmdArgs) error {
 		logger.Error(ipamErr)
 	}
 
-	calicoClient, err := CreateClient(conf)
-	if err != nil {
-		return err
-	}
-
-	ep := api.WorkloadEndpointMetadata{
-		Name:         args.IfName,
-		Node:         nodename,
-		Orchestrator: orchestrator,
-		Workload:     workload}
 	if err = calicoClient.WorkloadEndpoints().Delete(ep); err != nil {
```

Comment:
I don't think we want to retry, I don't see where/why we need to do that. I'm logging it and returning the error in that case.